python3Pyside2QML/src/main.py

```diff
@@ -8,16 +8,7 @@
 if __name__ == "__main__":
     print("Hello Torizon!")
 
-    qurlPath = ""
-    if(os.path.exists(os.path.expanduser("~/app/mainwindow.qml"))):
-        # Path of the qml source code file inside the debug container on the board, to debug the application remotelly on the board
-        qurlPath = os.path.expanduser("~/app/mainwindow.qml")
-    elif (os.path.exists("/app/mainwindow.qml")):
-        # Path of the qml source code file inside the release container on the board, to run the release (production) application on the board
-        qurlPath = "/app/mainwindow.qml"
-    elif(os.path.exists("src/mainwindow.qml")):
-        # Path of the qml source code file on host, to debug the application locally
-        qurlPath = "src/mainwindow.qml"
+    qurlPath = qurlPath = "src/mainwindow.qml"
```

Comment:
this also works when running from the image generated from production Dockerfile?

Reply:
Actually it had a little issue there. I fixed it in the latest rebase now.